### PR TITLE
Adding openssh tests

### PR DESCRIPTION
**Pull request by nasastry**
security/openssh-tests.py: adding openssh unit tests

Signed-off-by: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by nasastry** [comment]
```
JOB ID     : 00a0a46a72ec4ce88cc71e28e4b0386d3af36fd9
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-29T08.22-00a0a46/job.log
 (1/1) openssh-tests.py:OpenSSH.test:  PASS (274.26 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2020-09-29T08.22-00a0a46/results.html
JOB TIME   : 275.13 s
```

**Comment by nasastry** [comment]
SLES:
```
JOB ID     : ab6eda56ffdfe43805db8aeb2f4c1631e1767aa5
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-06T09.22-ab6eda5/job.log
 (1/1) openssh-tests.py:OpenSSH.test: PASS (48.98 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2020-10-06T09.22-ab6eda5/results.html
JOB TIME   : 50.00 s
```

RHEL:
```
JOB ID     : 8454bf2a98aa09f21e54cc868cb524d4647cba4b
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2020-10-02T13.09-8454bf2/job.log
 (1/1) openssh-tests.py:OpenSSH.test: PASS (252.58 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2020-10-02T13.09-8454bf2/results.html
JOB TIME   : 252.64 s
```

File: security/openssh-tests.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    openssh-testsuite
+    :avocado: tags=security,testsuite
+    """
+    def setUp(self):
+        '''
+        Install the basic packages to support openssh
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        deps = ['gcc', 'make', 'autoconf', 'libedit-devel', 'pam-devel']
```
Comment (harish-24) [inline security/openssh-tests.py:34]: Same here, As we are adding packages to generic list make sure it is handled for Debian based distros.

**Comment by nasastry** [comment]
@harish-24 Taken care of Ubuntu:

Ubuntu:
```
JOB ID     : e499a45ce8b66cd6f6eeb4c449e9ce444461e362
JOB LOG    : /root/avocado/job-results/job-2020-10-05T10.21-e499a45/job.log
 (1/1) openssh-tests.py:OpenSSH.test: CANCEL: Ubuntu not supported for this test (0.00 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB TIME   : 0.33 s
```


RHEL:
```
Fetching asset from openssh-tests.py:OpenSSH.test
JOB ID     : 1712ca001964982c42f5900a924973be1db0b340
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2020-10-05T11.22-1712ca0/job.log
 (1/1) openssh-tests.py:OpenSSH.test: PASS (252.54 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2020-10-05T11.22-1712ca0/results.html
JOB TIME   : 252.66 s
```

**Review by harish-24 — APPROVED** [review]
LGTM

**Comment by harish-24** [comment]
Can be merged on fixing the travis issue.

**Pull request merged**